src/fromager/sdist.py

```diff
@@ -311,7 +311,7 @@ def download_wheel(
     wheel_filename = output_directory / os.path.basename(urlparse(wheel_url).path)
     if not wheel_filename.exists():
         logger.info(f"{req.name}: downloading pre-built wheel {wheel_url}")
-        wheel_filename = _download_wheel_check(output_directory, wheel_url)
+        wheel_filename = _download_wheel_check(output_directory, wheel_url, ctx)
```

Comment:
We have the context as the first argument to functions everywhere else, could you follow that pattern here, too?

Reply:
okay, done!